### PR TITLE
Glenaeon board meeting

### PR DESCRIPTION
**Pull request by paked**

### CODE REVIEW

**Comment by paked** [comment]
Fixing this now.

**Comment by MaxWofford** [comment]
@paked Awesome contribution! Feel free to assign someone if this is ready for merge.

**Comment by paked** [comment]
@MaxWofford done :)

File: case_studies/glenaeon_rudolf_steiner_school/03_board_meeting/recap.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+In Australia we don't have the "Club Culture" that exists within America. For
+us, attending a club is the exception, not the rule. Thus, it has taken a bit
+more work than orginally anticipated to get our club set up.
+
+## The Proposal
+
+One of the things I was asked to do was write a short proposal for our club.
+
+> # Club Proposal
```
Comment (MaxWofford) [inline case_studies/glenaeon_rudolf_steiner_school/03_board_meeting/recap.md]: I know it's a quote, but would you please add a newline after this header?

Comment (paked) [inline case_studies/glenaeon_rudolf_steiner_school/03_board_meeting/recap.md]: @MaxWofford pushed

**Comment by MaxWofford** [comment]
@paked This looks great! Nice job with the proposal

**Pull request closed without merging**